commit 2ccedb87bce726c8321a7f766ed696a3d026e34f (refs/changes/75/5275/1)
Author: Josh Wilsdon <josh@wilsdon.ca>
Date:   2018-12-20T12:43:17-08:00 (10 months ago)
    
    TRITON-1055 sdcadm experimental update-agents with --just-download flag fails on SHA1

diff --git a/lib/cli/do_update_agents.js b/lib/cli/do_update_agents.js
index 226c7b5..45315b0 100644
--- a/lib/cli/do_update_agents.js
+++ b/lib/cli/do_update_agents.js
@@ -707,6 +707,11 @@ UpdateAgents.prototype.exec = function exec(callback) {
                 ctx.absfname
             ];
 
+            if (self.justDownload) {
+                next();
+                return;
+            }
+
             common.execFilePlus({
                 argv: argv,
                 log: self.sdcadm.log
diff --git a/package.json b/package.json
index c5745f7..b5b4ea0 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "sdcadm",
   "description": "Administer a SmartDataCenter (SDC) standup",
-  "version": "1.24.1",
+  "version": "1.24.2",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
